pkg/machines/xml-helpers/array-helper.js

```diff
@@ -0,0 +1,9 @@
+// This is copied from https://github.com/nashwaan/xml-js/blob/master/lib/array-helper.js
+
+export function isArray(value) {
+    if (Array.isArray) {
+        return Array.isArray(value);
+    }
+    // fallback for older browsers like  IE 8
+    return Object.prototype.toString.call(value) === '[object Array]';
```

Comment:
We don't support these, so the whole helper function can be dropped. We can rely on `Array.isArray()`.

Reply:
I removed completely the xml helpers.. In the end I think just parsing the DOMParser will do just fine. Sorry for the noise with xmljs.